pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "multiversx-sdk-cli"
-version = "10.1.0"
+version = "10.1.1"
```

Comment:
Debatable - maybe minor version, since it's a theoretical breaking change in the output format.

Reply:
Let's have it as patch, as it is :+1: